Samples/WinMLSamplesGallery/WinMLSamplesGallery/Pages/Benchmark.xaml

```diff
@@ -0,0 +1,14 @@
+﻿<Page

```

Comment:
what is this page for?

Reply:
Seemed like a new page @smk2007 was going to add. Deleting for now.